ramls/schema/loan-policy-id.json

```diff
@@ -7,6 +7,24 @@
     "loanPolicyId": {
       "description": "ID of the loan policy determined by the circulation rules",
       "type": "string"
+    },
+    "appliedRuleConditions": {
+      "description": "Addition information about conditions which were present in the applied rule",
+      "type": "object",
+      "properties": {
+        "isItemTypePresent": {
```

Comment:
Hmm, thinking about this, does `itemTypeMatched` read better to you?

I'm conscious this is a public API and so needs to be more carefully designed. Apologies if my previous suggestion wasn't good :-/

Reply:
Also, what is `ItemType` anyway? It appears to be material type. Unless we are moving away from "material type" (which I am not aware of) in circulation, why don't we use `itemMaterialTypeMatched` or `isItemMaterialTypePresent` or simply `materialTypeMatched` or `isMaterialTypePresent`?

The story mentions "Item Type" once, which I think was a typo since "Material Type" is mentioned 61 times (including comments). Then again, I might just be out of the loop here...

Reply:
> Also, what is `ItemType` anyway? It appears to be material type. Unless we are moving away from "material type" (which I am not aware of) in circulation, why don't we use `itemMaterialTypeMatched` or `isItemMaterialTypePresent` or simply `materialTypeMatched` or `isMaterialTypePresent`?

I don't believe we are moving away from material type. Item type was the name used for the parameter at the beginning, and we've not had chance to fix it (it would be a breaking change, unless we support both).

We could name this new property `materialTypeMatches` or similar. The challenge comes with explaining the discrepancy between it and the input parameter.

> The story mentions "Item Type" once, which I think was a typo since "Material Type" is mentioned 61 times (including comments). Then again, I might just be out of the loop here...

I don't know whether the story is using item type because there is a synonymous domain term for material type, or whether the misnamed parameter has bled back into the requirements language.

Reply:
Ah... I didn't realize this was prehistoric stuff. 😄 I just looked at `ItemType` and the comment says:
> Store the UUID of the item's type (the material type).

It is a shame we deviated from the existing item domain term to some less obvious circulation domain term. I suppose it is what it is now... 😞

Reply:
I used ItemType because Drools returns the condition as "ItemType". 
Renamed the properties in my last commit: materialTypeMatch, loanTypeMatch, patronGroupMatch